tee-worker/ts-tests/integration-tests/identity.test.ts

```diff
@@ -22,9 +22,34 @@ import { assert } from 'chai';
 import { multiAccountTxSender, sendTxsWithUtility } from './common/transactions';
 import type { LitentryPrimitivesIdentity } from 'sidechain-api';
 import type { LitentryValidationData, Web3Network } from 'parachain-api';
-import type { TransactionSubmit } from './common/type-definitions';
+import type { IntegrationTestContext, TransactionSubmit } from './common/type-definitions';
 import type { HexString } from '@polkadot/util/types';
 import { ethers } from 'ethers';
+import { sendRequest } from './common/call';
+import * as base58 from 'micro-base58';
+import { decodeRpcBytesAsString } from './common/call';
+
+async function getWorkerAddress(context: IntegrationTestContext): Promise<string> {
+    const requestAcc = { jsonrpc: '2.0', method: 'author_getEnclaveId', params: [], id: 1 };
+    const resAcc = await sendRequest(context.tee, requestAcc, context.api);
```

Comment:
Would you like to be consistent with other impl?
Like 
```
function getEnclaveSignerAccount ....
const request = ...
const res = ...
const enclaveSignerAccount = ...
```

@grumpygreenguy might have better ideas

Reply:
My 2 picoLIT: it should do what it says and say what it does :D And what it does is just calling the other method so I think the name should reflect just that. Assuming it's `enclave_signer_public_key`, then I'd expect something like

```
async function getEnclaveSignerPublicKey(...) {
    const request = { ... }
    const response = await sendRequest(...)
    if (!response.status.isOk) {
        throw new Error(...)
    return response.value
}
```

(on that note: as a reader, without any additional context, I have no idea what the `Acc` in `requestAcc` is supposed to stand for; there's no other request/response pair in the function body tho, so it can just be left as `request` and `response`).